code_snippets/java/build.gradle.kts

```diff
@@ -0,0 +1,67 @@
+import com.google.protobuf.gradle.id
+
+plugins {
+  java
+  application
+
+  id("com.google.protobuf") version "0.9.1"
+}
+
+repositories {
+  mavenCentral()
+}
+
+val restateVersion = "0.8.0"
```

Comment:
For updating the versions i guess we can reuse pretty much the same automation we have in examples repo, which are the update scripts here https://github.com/restatedev/examples/tree/main/.tools and https://github.com/restatedev/examples/blob/main/.github/workflows/pre-release.yml